s3fs/core.py

```diff
@@ -118,10 +118,13 @@ async def _error_wrapper(func, *, args=(), kwargs=None, retries):
         except ClientError as e:
             logger.debug("Client error (maybe retryable): %s", e)
             err = e
+            wait_time = asyncio.sleep(min(1.7**i * 0.1, 15))
```

Comment:
I'm pretty sure you will see a warning if you create this coroutine, but none of the following conditions apply, so you never wait on it

Reply:
sorry, right, let me update it real quick